apps/web/src/components/common/PromoBanner/PromoBanner.tsx

```diff
@@ -0,0 +1,104 @@
+import css from './styles.module.css'
+import { Box, Button, Card, IconButton, Stack, Typography } from '@mui/material'
+import Image, { type StaticImageData } from 'next/image'
+import Link, { type LinkProps } from 'next/link'
+import CloseIcon from '@mui/icons-material/Close'
+import Track from '@/components/common/Track'
+import type { ReactNode } from 'react'
+import type { AnalyticsEvent } from '@/services/analytics'
+
+export interface PromoBannerProps {
+  title: string
+  description?: string
+  ctaLabel: string
+  href: LinkProps['href']
+  trackOpenProps: AnalyticsEvent
+  trackHideProps: AnalyticsEvent
+  onDismiss?: () => void
+  imageSrc?: string | StaticImageData
+  imageAlt?: string
+  endIcon?: ReactNode
+  customClass?: string
+  customBackground?: string
+}
+
+export const PromoBanner = ({
+  title,
+  description,
+  ctaLabel,
+  href,
+  onDismiss,
+  imageSrc,
+  imageAlt,
+  endIcon,
+  trackOpenProps,
+  trackHideProps,
+  customClass,
+  customBackground,
+}: PromoBannerProps) => {
+  return (
+    <Card
+      className={`${css.banner} ${customClass || ''}`}
+      style={{ borderRadius: '12px' }}
+      sx={customBackground ? { background: customBackground } : undefined}
+    >
+      <Stack
+        direction={{ xs: 'row', md: 'row' }}
+        alignItems="center"
+        spacing={2}
+        sx={{ display: 'inline-flex', flexWrap: 'nowrap' }}
+      >
+        {imageSrc ? (
+          <Image className={css.bannerImage} src={imageSrc} alt={imageAlt || ''} width={95} height={95} />
+        ) : null}
+        <Box>
+          <Typography variant="h4" fontWeight="bold" color="common.white" className={css.bannerText}>
+            {title}
+          </Typography>
+
+          {description ? (
+            <Typography variant="body2" color="primary.light" sx={{ pt: 0.5 }} className={css.bannerText}>
+              {description}
+            </Typography>
+          ) : null}
+
+          <Track {...trackOpenProps}>
+            <Link href={href} passHref>
+              <Button
+                {...(endIcon && { endIcon })}
+                variant="text"
+                size="compact"
+                sx={{ mt: 0, p: 0, pt: 1, color: 'common.white' }}
```

Comment:
I think this button needs a bit of a top margin. You can also see the text is too dim in the HTML version.

Compare Figma:
<img width="746" height="152" alt="Screenshot 2025-11-06 at 08 00 30" src="https://github.com/user-attachments/assets/0a431678-17c1-4af1-81e6-cc0a43fb7652" />

Storybook:
<img width="798" height="158" alt="Screenshot 2025-11-06 at 08 00 50" src="https://github.com/user-attachments/assets/31d9cb48-15f7-4c74-923e-07f443969f9d" />

Reply:
The font color is good now but the Learn more link still needs a bigger top margin.

<img width="962" height="168" alt="Screenshot 2025-11-06 at 10 49 12" src="https://github.com/user-attachments/assets/835078c8-28ca-4a44-b55f-41922b3779b3" />